app/src/main/java/com/example/bigowlapp/viewModel/ScheduleListViewModel.java

```diff
@@ -14,15 +14,19 @@ public ScheduleListViewModel() {
         // used implicitly when ViewModel constructed using ViewModelProvider
     }
 
-    public LiveData<List<Schedule>> getScheduleList(String groupID) {
+    public LiveData<List<Schedule>> getScheduleList(Boolean isGroupSupervisor, String groupID) {
```

Comment:
You should use `boolean` instead of `Boolean`.

Sonarqube considers it a code smell.